Tests/SonarQube.MSBuild.Tasks.IntegrationTests/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -91,8 +91,7 @@ public void Roslyn_Settings_ValidSetup_MergeWithExistingSettings_AbsolutePath()
             AssertExpectedAdditionalFiles(result, GetDummySonarLintXmlFilePath());
 
             string mergedRulesetFilePath = AssertMergedResultFileExists(result, sourceRulesetFilePath);
-            RuleSetAssertions.AssertExpectedIncludeFiles(mergedRulesetFilePath, "c:\\existing.ruleset");
-            RuleSetAssertions.AssertExpectedIncludeAction(mergedRulesetFilePath, "c:\\existing.ruleset", RuleSetAssertions.DefaultActionValue);
```

Comment:
Now we only ever expect the Include action to be "Warning", so I've updated *AssertExpectedIncludeFiles* to check this too and removed *AssertExpectedIncludeAction*.